src/components/Tags/index.jsx

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+
+import * as styles from "./index.module.scss"
+
+function Tags({ tags }) {
+  return (
+    <ul className={styles.tags}>
+      {tags ? (
```

Comment:
What happens to this component if there are no `tags`? Should we assume that if we're using this component there are always `tags`?